File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentRepository.java
package com.metaengine.demo.shipping.services;

import java.util.List;

import com.metaengine.demo.shipping.aggregates.Shipment;

/** ShipmentRepository for the shipping domain. */
public class ShipmentRepository {

    /** Persists a new Shipment from the partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Finds a Shipment by id. May return null. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Lists Shipments up to the given limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Deletes the Shipment with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/a-mcp/output/shipping/services/shipment_repository.py
from typing import List, Optional
from shipping.aggregates.shipment import Shipment

"""ShipmentRepository service."""
class ShipmentRepository:
    def create(self, input: Shipment) -> Shipment:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Shipment]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> List[Shipment]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

